cmd/application/conns_test.go

```diff
@@ -139,6 +160,61 @@ func TestConnPrintAndReset(t *testing.T) {
 	connManager.connStats.PrintAndReset(logger)
 }
 
+func TestConnHandleNewTCPConnNoData(t *testing.T) {

```

Comment:
why don't we remove this test since it doesn't actually test the bug that is fixed by this PR and might add to the confusion in the future.